portfolio/src/main/webapp/contact.html

```diff
@@ -17,10 +17,23 @@
             </ul>
         </nav>
         
-    <div id="content">
+    <div id = "content">
+        <section class = "socials">
+
+        </section>
+
+        <hr>
```

Comment:
I believe <hr> tags are generally discouraged in favor or css styling. Try adjusting the bottom padding or margin on the above section, or the top on the below section.